test/functional/p2p_invalid_tx.py

```diff
@@ -157,11 +157,11 @@ def run_test(self):
 
         # restart node with sending BIP61 messages disabled, check that it disconnects without sending the reject message
         self.log.info('Test a transaction that is rejected, with BIP61 disabled')
```

Comment:
> self.log.info('Test a transaction that is rejected, with BIP61 disabled')

Is it possible to disable BIP61?

Reply:
no. I removed the comment and changed the test.